src/app/app-routing.module.ts

```diff
@@ -1,19 +1,19 @@
 import { NgModule } from '@angular/core';
-import {RouterModule, Routes} from "@angular/router";
-import {HomeComponent} from "./home/home.component";
-import {SearchComponent} from "./search/search.component";
-import {MedicalInstitutionsComponent} from "./medical-institutions/medical-institutions.component";
-
+import { RouterModule, Routes } from '@angular/router';
+import { HomeComponent } from './features/home/home.component';
+import { MedicalInstitutionsModule } from './features/medical-institutions/medical-institutions.module';
 
 const routes: Routes = [
   { path: '', component: HomeComponent },
-  { path: 'medical-institutions', component: SearchComponent},
-  { path: 'medical-institutions/current-location', component: MedicalInstitutionsComponent},
-  { path: 'medical-institutions/address', component: MedicalInstitutionsComponent},
+  {
+    path: 'medical-institutions',
+    loadChildren: () =>
+      import('./features/medical-institutions/medical-institutions.module').then((m) => m.MedicalInstitutionsModule),
+  },
 ];
 
 @NgModule({
-  imports: [RouterModule.forRoot(routes)],
-  exports: [RouterModule]
+  imports: [RouterModule.forRoot(routes), MedicalInstitutionsModule],
```

Comment:
```suggestion
  imports: [RouterModule.forRoot(routes)],
```
[この記事](https://qiita.com/Yamamoto0525/items/e870713d9d05d2d36a80#%E3%83%AB%E3%83%BC%E3%83%88%E3%83%91%E3%82%B9%E3%81%B8%E3%81%AE%E3%83%AB%E3%83%BC%E3%83%86%E3%82%A3%E3%83%B3%E3%82%B0)を参考に書き換えてみたら直りました
ルートレベルのルーティングを定義するRouterModule.forRoot(routes)とMedicalInstitutionsModuleをimportsに入れた結果、MedicalInstitutionsModuleのルーティングが優先されてしまってたようです

Reply:
@tanimuranaomichi 
ありがとうございます！無事動作できました！